docs/examples/redshift.rst

```diff
@@ -0,0 +1,66 @@
+Redshift Example Project
+========================
+
+This branch provides a small collection of generalized pipelines for the training and utilization of red-shift estimation models. This branch is designed to allow simple use by only requiring that the configuration parameters of individual nodes be defined where necessary. The most involved alterations that should be necessary for most users is the definition of additional architectures in the **Resources** tab. It should be noted that any newly defined architecture should have an output length and input shape that match the *num_bins* and *input_shape* configuration parameters being used in the various pipelines.
```

Comment:
- introduce/link to the project. 
- "This branch" -> I don't believe it is on a branch anymore, is it?
- "red-shift" -> "redshift"

Reply:
Link added to examples repo. More specific link will be added later as with the other example link discussed above.

You are correct, that is an oversight from adapting an earlier version of the tutorial.